inventory-generation/notifications/inventory/hosts

```diff
@@ -0,0 +1,2 @@
+[local]
```

Comment:
Can we come up with a more descriptive name than `local` for this? Perhaps `notification_host`?

Reply:
Updated to `notification_host`